src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateFromForm = parseDate(dateFrom);
+        LocalDate dateToForm = parseDate(dateTo);
```

Comment:
The variable names 'dateFromForm' and 'dateToForm' could be more informative. Consider renaming them to 'startDate' and 'endDate' to reflect their purpose in the context of the method.